core/notfound.ml

```diff
@@ -53,6 +53,15 @@ end
 module List =
 struct
   include List
+
+  let find2 p l =
```

Comment:
This doesn't seem to be used anywhere (and doesn't seem to do anything find doesn't).

Reply:
Can be deleted now.